blocks/article-tag-block/features/tag/default.test.jsx

```diff
@@ -1,144 +1,148 @@
-// 			expect(wrapper.children().find(".c-pill").at(0).props().href).toBe("/tags/dogs%20slug/");
-// 			expect(wrapper.children().find(".c-pill").at(1).props().href).toBe("/tags/cats%20slug/");
-// 		});
-// 	});
-
-// 	describe("when the global content has a property called taxonomy which contains tags array with out slugs", () => {
-// 		afterEach(() => {
-// 			jest.resetModules();
-// 		});
-
-// 		beforeEach(() => {
-// 			useFusionContext.mockReturnValue({
-// 				arcSite: "the-sun",
-// 				globalContent: {
-// 					taxonomy: {
-// 						tags: [
-// 							{
-// 								description: "dogs",
-// 								text: "dogs text",
-// 							},
-// 							{
-// 								description: "cats",
-// 								text: "cats text",
-// 							},
-// 						],
-// 					},
-// 				},
-// 			});
-// 		});
-
-// 		it("should render pill components", () => {
-// 			const wrapper = mount(<ArticleTags />);
-// 			const tags = wrapper.find(".c-pill");
-// 			expect(tags.length).toBe(2);
-// 		});
-// 	});
-
-// 	describe("when the global content has an empty tags array", () => {
-// 		afterEach(() => {
-// 			jest.resetModules();
-// 		});
-
-// 		beforeEach(() => {
-// 			useFusionContext.mockReturnValue({
-// 				arcSite: "the-sun",
-// 				globalContent: {
-// 					taxonomy: {
-// 						tags: [],
-// 					},
-// 				},
-// 			});
-// 		});
-
-// 		it("should not render anything", () => {
-// 			const wrapper = mount(<ArticleTags />);
-// 			expect(wrapper.isEmptyRender()).toBe(true);
-// 		});
-// 	});
-
-// 	describe("when the global content does not have a taxonomy property", () => {
-// 		afterEach(() => {
-// 			jest.resetModules();
-// 		});
-
-// 		beforeEach(() => {
-// 			useFusionContext.mockReturnValue({
-// 				arcSite: "the-sun",
-// 				globalContent: {},
-// 			});
-// 		});
-
-// 		it("should not render anything", () => {
-// 			const wrapper = mount(<ArticleTags />);
-// 			expect(wrapper.isEmptyRender()).toBe(true);
-// 		});
-// 	});
-// });
+	describe("when the global content has an empty tags array", () => {
+		afterEach(() => {
+			jest.resetModules();
+		});
+
+		beforeEach(() => {
+			useFusionContext.mockReturnValue({
+				arcSite: "the-sun",
+				globalContent: {
+					taxonomy: {
+						tags: [],
+					},
+				},
+			});
+		});
+
+		it("should not render anything", async () => {
+			render(<ArticleTags />);
+			const wrapper = await screen.findByRole("generic");
+			expect(wrapper).toContainHTML('<div />');
+		});
+	});
+
+	describe("when the global content does not have a taxonomy property", () => {
+		afterEach(() => {
+			jest.resetModules();
+		});
+
+		beforeEach(() => {
+			useFusionContext.mockReturnValue({
+				arcSite: "the-sun",
+				globalContent: {},
+			});
+		});
+
+		it("should not render anything", async () => {
+			const { container } = render(<ArticleTags ItemLength={1} />);
+			// eslint-disable-next-line testing-library/no-node-access
+			expect(container.firstChild).toBeNull();
```

Comment:
Another place you can use `expect(container).toBeEmptyDOMElement();`. Remove the eslint disable as well.